src/CalcViewModel/DateCalculatorViewModel.cpp

```diff
@@ -60,8 +60,7 @@ DateCalculatorViewModel::DateCalculatorViewModel()
     auto today = calendar->GetDateTime();
 
     // FromDate and ToDate should be clipped (adjusted to a consistent hour in UTC)
```

Comment:
>(adjusted to a consistent hour in UTC) [](start = 45, length = 38)

I believe this comment is no longer correct. Mind fixing it?

Reply:
It's still the case, `FromDate` and `ToDate` must be clipped to midnight UTC time.